content/en/monitors/service_level_objectives/_index.md

```diff
@@ -85,6 +83,50 @@ This can result in the overall uptime being lower than the average of the indivi
 
 You can view, edit your SLO and its properties and see the status over time and the history of your SLO from the [SLO status page][1].
 
+### SLO Default View
+
+The default SLO view is loaded when you land on the SLO list view. 
+
+The default view includes:
+
+- An empty search query
+- A list of all defined SLOs in your organization 
+- A list of available facets in left side facet list
+
+### Saved Views
+
+Saved views allow you to save and share across your organization search customizations in the SLO list view for SLOs that are most relevant for you including:
```

Comment:
Saved views allow you to save and share customized searches in the SLO list view for SLOs that are most relevant for you including:

Sound better?

Reply:
I think so